### PR TITLE
refactor: deprecated field hidden

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Related Issues

Closes to #1345.

### Description and Example

The deprecated field is hidden.

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: android/beagle/src/main/java/br/com/zup/beagle/android/components/TextInput.kt [hunk 1]
```diff
@@ -72,6 +72,7 @@ data class TextInput(
     val disabled: Bind<Boolean>? = null,
     val readOnly: Bind<Boolean>? = null,
     val type: Bind<TextInputType>? = null,
+    @Deprecated("It was deprecated in version 1.6.0 and will be removed in a future version. Use field display to control visibility.")
```
Comment (botbeaglezup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/components/TextInput.kt:75]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/e831d02c4f328cd084896f70d16712edbf9d5e29/android/beagle/src/main/java/br/com/zup/beagle/android/components/TextInput.kt#L75"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>

File: android/beagle/src/main/java/br/com/zup/beagle/android/components/TextInput.kt [hunk 2]
```diff
@@ -81,6 +82,7 @@ data class TextInput(
     val onBlur: List<Action>? = null,
 ) : InputWidget() {
 
+    @Deprecated("It was deprecated in version 1.6.0 and will be removed in a future version. Use field display to control visibility.")
```
Comment (botbeaglezup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/components/TextInput.kt:85]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/e831d02c4f328cd084896f70d16712edbf9d5e29/android/beagle/src/main/java/br/com/zup/beagle/android/components/TextInput.kt#L85"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>

File: backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/ui/TextInput.kt [hunk 3]
```diff
@@ -52,6 +52,7 @@ data class TextInput(
     val disabled: Bind<Boolean>? = null,
     val readOnly: Bind<Boolean>? = null,
     val type: Bind<TextInputType>? = null,
+    @Deprecated("It was deprecated in version 1.6.0 and will be removed in a future version. Use field display to control visibility.")
```
Comment (botbeaglezup) [inline backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/ui/TextInput.kt:55]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/e831d02c4f328cd084896f70d16712edbf9d5e29/backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/ui/TextInput.kt#L55"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>

File: backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/ui/TextInput.kt [hunk 4]
```diff
@@ -60,6 +61,34 @@ data class TextInput(
...
+        type = valueOfNullable(type),
+        error = valueOfNullable(error),
+        showError = valueOfNullable(showError),
+        styleId = styleId,
+        onChange = onChange,
+        onFocus = onFocus,
+        onBlur = onBlur
+    )
+
+    @Deprecated("It was deprecated in version 1.6.0 and will be removed in a future version. Use field display to control visibility.")
```
Comment (botbeaglezup) [inline backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/ui/TextInput.kt:91]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/e831d02c4f328cd084896f70d16712edbf9d5e29/backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/ui/TextInput.kt#L91"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>

File: iOS/Sources/Beagle/Sources/Components/Widget/TextInput/Tests/TextInputTests.swift [hunk 5]
```diff
@@ -46,6 +46,7 @@ class TextInputTests: XCTestCase {
     func testTextInputTypes() {
         // Given
         let component = TextInput(value: "", placeholder: "", type: .value(.password))
+
```
Comment (botbeaglezup) [inline iOS/Sources/Beagle/Sources/Components/Widget/TextInput/Tests/TextInputTests.swift:50]: <table data-meta="generated_by_ios">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/e9ca91f2ed61c70763e0982f916c50f4e0870471/iOS/Sources/Beagle/Sources/Components/Widget/TextInput/Tests/TextInputTests.swift#L50"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>TextInputTests.swift:50</code></td>
    </tr>
  </tbody>
</table>

**Review by matheusribeirozup — APPROVED** [review]

**Review by luisgustavozup — APPROVED** [review]

**Review by viniciusguardieirozup — APPROVED** [review]

**Pull request merged**